calico_containers/calicoctl.py

```diff
@@ -57,12 +57,12 @@
 from netaddr.core import AddrFormatError
 from prettytable import PrettyTable
 
-from calico_containers.adapter.datastore import (ETCD_AUTHORITY_ENV,
```

Comment:
I've modified it so that calicoctl.py imports `pycalico.datastore` dropping the reference to `calico_containers` in the module path, which is odd.

In order for pyinstaller to build `calicoctl` correctly I had to set the PYTHONPATH for the build. command.